File: Latihan 3/C++/cicilan.cpp
#include <iostream>

using namespace std;

int main() {
    int harga_motor, lama_kredit;

    cout << "Masukan harga motor                = ";
    cin >> harga_motor;
    cout << "Masukan lama kredit (dalam bulan)  = ";
    cin >> lama_kredit;

    int harga_pokok = harga_motor / lama_kredit;
    int bunga = (harga_pokok * 0.1);
    int cicilan = harga_pokok + bunga;
    int total_harga = cicilan * lama_kredit;
    int keuntungan = total_harga - harga_motor;

    cout << "Cicilan /bulan     : " << cicilan << endl;
    cout << "Total harga motor  : " << total_harga << endl;
    cout << "Keuntungan Dealer  : " << keuntungan;

    return 0;

}

File: Latihan 3/Python/cicilan.py
harga_motor = int(input("Masukan Harga Motor : "))
lama_kredit = int(input("Masukkan lama kredit (dalam bulan): "))

harga_pokok = int(harga_motor / lama_kredit)
bunga = int(harga_pokok * 0.1)
cicilan = int(harga_pokok + bunga)
total_harga = int(cicilan * lama_kredit)
keuntungan = int(total_harga - harga_motor)

print("Harga Pokok = ", harga_pokok)
print("Bunga = ", bunga)
print("Cicilan /bulan = ", cicilan)
print("Total harga motor = ", total_harga)
print("Keuntungan Dealer = ", keuntungan)

